### PR TITLE
updated the gaussMap function

### PR DESCRIPTION
**Pull request by ianmseddy**
Updated the gaussMap function to include possibility of Gaussian models other than the current exponential model.

### CODE REVIEW

File: R/initialize.R [hunk 1]
```diff
@@ -17,10 +17,18 @@ if (getRversion() >= "3.1.0") {
...
+#' @param method   The type of model used to produce the Gaussian pattern. Accepted
+#'                 arguments include 'RMgauss' (Gaussian covariance model), 'RMstable'
+#'                 (the stable powered exponential model), and the default, "RMexp"
+#'                 (exponential covariance model)
+#'
 #' @param speedup  An numeric value indicating how much faster than 'normal'
 #'                 to generate maps. It may be necessary to give a value larger
 #'                 than 1 for large maps. Default is 1.
 #'
+#' @param alpha    a required parameter of the 'RMstable' model, should be in the interval
```
Comment (achubaty) [inline R/initialize.R]: I would just add a `...` argument to pass through to the chosen method.

Comment (ianmseddy) [reply R/initialize.R]: Thanks Alex. Eliot just walked me through the process of documentation, testing, adding examples. I  will give this a shot, but expect more rejections!

**Comment by achubaty** [comment]
is this the same as #20 ?

These PRs should have been made on the `development` branch. Please change the branch and rerun tests.

**Comment by ianmseddy** [comment]
I accidentally made a new pull request instead of just editing the old one (that was missing the documentation files). Also, it was to the master branch and not the development. I'll fix this.

**Pull request closed without merging**